test/core/requestExecutioner.test.ts

```diff
@@ -30,12 +30,10 @@ describe("Can return the output of the command", () => {
       useLiveLog: false
     };
     let commandRunner = new CommandRunner(mockedCommandRunnerOptions);
-    let payload = `{ "warnings": ["boop", "beep", "bop"] }`;
-    expect(
-      await commandRunner.runCommand(
-        `>&2 echo ${platform() === "win32" ? payload : payload.replace(/"/g, '\\"')}`
-      )
-    ).toContain("Completed with warnings");
+    // warningTest.js is a simple Node two stringified JSON, one into stdout, the other into stderr.
```

Comment:
two ?

Reply:
2 is the error stream on bash and PowerShell.